docs/tutorials/comparison.md

```diff
@@ -79,302 +123,753 @@ source1 = "FirstSourceName"
-- Untick `I` so only `P` and `B` are selected, or only select `B` if you want matching frame-types.
-- Tick the `public` box to make the comparison available on the slow.pics homepage.
-- Name the collection appropriately for Slow.pics, get the TMDB ID from <https://www.themoviedb.org/> and paste it in the TMDB box, with it correctly set to Movie/TV depending on the content.
-- Set the tag to the same name as listed on TMDB.
-- Set "Delete After" to your desired value, by default comps expire after 2 years of not being viewed.
-- Under `random` enter a frame count amount, you should use a large number of images to make the comparison as effective as possible, ideally at least 40 since not all frames will be useful.
-- Make sure your PNG compression is set to 0 (Max) under Settings -> Main*
-- Hit `Start Upload` and patiently wait while VS-Preview collects the frames and uploads the comparison.
+When creating your comparison, we recommend naming it with the show title and sources used. Some examples are:
 
-**If you wish to upload to a slow.pics account for comp history and ability to edit comps, do the following:**
+- By video source: `The Eminence in Shadow - BD vs. WEB`
+- By release type: `Watashi no Oshi wa Akuyaku Reijou. - Minis vs. Streams`
+- By release group: `Miss Kobayashi's Dragon Maid - Okay-Subs vs. Beatrice-Raws`
 
-- Make sure you're logged into slow.pics
-- Open the developer menu, go to Application at the top, then Storage on the left, open Local Storage and select slow.pics. Copy the browserId.
-- Still in the developer menu, go to Network, refresh the page to populate the list, select the slow.pics entry and scroll until you see SLPSESSION and copy the ID.
-- Back in VS-Preview, go to settings, Comp, and paste the two IDs in the relevant boxes.
+==- 2 - Framing parameters
 
-+++ Semi-auto
+#### Random
 
-### Marking the frames
+The *Random* parameter sets the number of frames to skip before screenshotting. *This is a setting used when creating [automatic](#automatic-badge-variant-secondary-text-fastest-option)/[semi-automatic](#semi-automatic-badge-icon-variant-primary-text-recommended) comparisons.*
 
-- Press `Shift` & `Right Arrow` keys to move forward a set amount of frames (configurable in playback section of the toolbar)
-- `Number` keys to switch between video sources and compare quality.
-- Press `Control` & `Space` keys to mark the frame numbers. (Configurable in cfg files)
-- Aim to screenshot a variety of scenes, light/dark, low/high motion, etc.
-- Try and match frame type when screenshotting, e.g. all sources on a `B` frame, single frame jump comes in handy for when they don't match (`Arrow keys`).
-- If a source file does not have `B` frames for you to match, you should skip matching frame type entirely for that source. This is usually true for Crunchyroll WEB-DLs, which have no `B` frames.
+For example, a value of `48` means VSPreview will take a screenshot for every 48 frames in all sources.
```

Comment:
this reads a bit confusing. should be something like "For example, a value of `48` means VSPreview will take 48 random screenshots from all the sources"